### ISSUE TITLE
Attempt to us expired subkey while valid key is present

### ISSUE THREAD

**Issue by vsaw** [mailvelope/mailvelope#428]
I try to send an encrypted message to a recipient with the following key "structure":

```
Type bits/keyID     cr. time   exp time   key expir
pub  4096R/AAAAAAAA 2014-01-04            

uid User Name <recipient@email.tld>
sig  sig3  AAAAAAAA 2014-01-04 __________ 2015-01-03 [selfsig]
sig  sig   BBBBBBBB 2014-04-07 __________ __________ Other User Signature <other.user@signature.email>
sig  sig3  AAAAAAAA 2015-01-03 __________ 2016-01-03 [selfsig]
sig  sig3  BBBBBBBB 2015-01-03 __________ __________ Other User Signature <other.user@signature.email>
sig  sig3  AAAAAAAA 2016-01-08 __________ 2017-01-09 [selfsig]

sub  4096R/CCCCCCCC 2014-01-04            
sig sbind  AAAAAAAA 2014-01-04 __________ 2015-01-03 []

sub  4096R/DDDDDDDD 2015-01-03            
sig sbind  AAAAAAAA 2015-01-03 __________ 2016-01-04 []
sig sbind  AAAAAAAA 2016-01-08 __________ 2017-01-09 []
```

However, looking at Mailvelope it shows me that subkey `DDDDDDDD` expires at 2016-01-04 and not on the 2017-01-09. When trying to encrypt a message Mailvelope tells me

> Could not encrypt this message: Error: Error encrypting message!

The key itself is working as the same key imported int GPG is able to encrypt files for the recipient.
# Versions
- Mailvelope: v1.5.2
- Chrome: 53.0.2785.92 (64-bit)

**Comment by nitrocode** [comment]
Yup. I'm getting the same issue with Chrome 56, Windows 7 64bit, using Mailvelope 1.7.1 while attempting to encrypt a message. I even tried disabling all extensions.

**Comment by toberndo** [comment]
Currently a limitation of OpenPGP.js: https://github.com/openpgpjs/openpgpjs/issues/266

**Comment by toberndo** [comment]
Should be fixed with the v2.0.0 release of Mailvelope which is now available.
